sdcm/cluster.py

```diff
@@ -3372,6 +3374,21 @@ def kill_gemini_thread(self):
                 if kill_result.exit_status != 0:
                     self.log.error('Terminate gemini on node %s:\n%s', loader, kill_result)
 
+    def create_swap_file(self):
+        swap_size = self.params.get("loader_swap_size", default=None)
+        if not swap_size:
+            self.log.info("Swap file is not configured")
+            return
+        commands = dedent("""sudo /bin/dd if=/dev/zero of=/var/swap.1 bs=1M count={}
```

Comment:
Those commands were tested on Ubuntu/Debian ?

Reply:
It's worth to test and make it work.

Currently we only use centos7 for loader & monitor?

Reply:
if it's not working on debian/ubuntu, we should put an if here and raise exception

Reply:
checked on 18.04. steps are working

Reply:
good